index.html

```diff
@@ -12,39 +12,175 @@
+            </ul>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+        <div class="box">
+            <img src="req/multi.png" alt="multi" class="banner">
+            <h3>Multi-Post
+                Stories</h3>
+                <ul>
+                    <li class="dot1">CANOPY</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot2">Back End Dev</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot3">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+        <div class="box">
+            <img src="req/tonic_2.png" alt="multi" class="banner">
+            <h3>Tonic</h3>
+                <ul>
+                    <li class="dot1">CANOPY</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot2">Back End Dev</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot3">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+        <div class="box">
+            <img src="req/multi_2.png" alt="multi" class="banner">
+            <h3>Multi-Post
+                Stories</h3>
+                <ul>
+                    <li class="dot1">CANOPY</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot2">Back End Dev</li>
+                    <li><img src="req/dot.png" alt="dot"></li>
+                    <li class="dot3">2015</li>
+                </ul>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul>
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+            </ul>
+            <input type="button" value="See Project">
+        </div>
+     </div>
+    </section>
+    <section>
+        <div class="about">
+            <h1 class="about-header">About<br>Myself</h1>
```

Comment:
- [ ] I can see that missed this prior to previous reviewers suggestion. It is imperative that we don't use more than 1 `h1` in our document. Please consider using other heading tags e.g. `h2`, `h3`, etc. This ensures that Search Engines don't get confused crawling our sites which may affect site's SEO ranking. Also, it is a conventional practice. 😉